apps/bfd-pipeline/bfd-pipeline-rda-grpc-server/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/DirectRdaLoadApp.java

```diff
@@ -18,25 +18,37 @@
  * Simple application that invokes an RDA API server and writes FISS claims to a database using
  * RdaLoadJob. This program was written for testing purposes and is not planned to be executed in a
  * production environment.
+ *
+ * <p>Two command line options are required when running the program:
+ *
+ * <ol>
+ *   <li>file: path to a properties file containing configuration settings
+ *   <li>claimType: either fiss or mcs to specify type of claims to download
+ * </ol>
  */
 public class DirectRdaLoadApp {
   private static final String RDA_PERSISTENCE_UNIT_NAME = "gov.cms.bfd.rda";
 
   public static void main(String[] args) throws Exception {
-    if (args.length != 1) {
-      System.err.printf("usage: %s configfile%n", DirectRdaLoadApp.class.getSimpleName());
+    if (args.length != 2) {
+      System.err.printf("usage: %s configfile claimType%n", DirectRdaLoadApp.class.getSimpleName());
       System.exit(1);
     }
     Properties props = new Properties();
     try (Reader in = new BufferedReader(new FileReader(args[0]))) {
       props.load(in);
     }
+    final String claimType = args[1];
+
     final RdaLoadOptions jobConfig = readRdaLoadOptionsFromProperties(props);
     final DatabaseOptions databaseConfig = readDatabaseOptions(props);
     final PipelineApplicationState appState =
         new PipelineApplicationState(
             new MetricRegistry(), databaseConfig, RDA_PERSISTENCE_UNIT_NAME);
-    final PipelineJob job = jobConfig.createFissClaimsLoadJob(appState);
+    final PipelineJob<?> job =
+        claimType.equals("fiss")
```

Comment:
is there another check to see if claimtype is "fiss" or "mcs" and if not raise an error?  If the intention of this app is ONLY for testing purposes than you can ignore, just want to make sure nothing happens in the future where "fiss" is mistyped (eg, "fis") and the mcs claims load runs

Reply:
Good point.  It is just for testing but it's easy enough to catch this.  I'll fix.